mkdocs.yml

```diff
@@ -86,8 +84,6 @@ nav:
       - ReadOnlyDomainControllerContainer: Setup/SetupAssist/ReadOnlyDomainControllerContainer.md
       - RebootPending: Setup/SetupAssist/RebootPending.md
     - SetupLogReviewer: Setup/SetupLogReviewer.md
-  - Transport:
-    - Compute-TopExoRecipientsFromMessageTrace: Transport/Compute-TopExoRecipientsFromMessageTrace.md
```

Comment:
These lines should not be removed.

Reply:
done, got the most recent mkdocs.yml file instead of the old one I had :)